cluster-provision/k8s/1.17/provision.sh

```diff
@@ -323,11 +330,19 @@ cp -rf /tmp/cnao/ /opt/
 for i in $(grep -A 2 "IMAGE" /opt/cnao/operator.yaml |grep value | awk '{print $2}'); do docker pull $i; done
 
 # Create a properly labelled tmp directory for testing
-mkdir -p /tmp/kubevirt.io/tests
-chcon -t container_file_t /tmp/kubevirt.io/tests
-echo "tmpfs /tmp/kubevirt.io/tests tmpfs rw,context=system_u:object_r:container_file_t:s0 0 1" >> /etc/fstab
+mkdir -p /provision/kubevirt.io/tests
```

Comment:
line 330 also have /tmp (cant put there a comment)
cp -rf /tmp/cnao/ /opt/

/lgtm cancel

Reply:
Sorry, I do not understand your comment.
- What "real place"?
- The cnao copy is moving files from /tmp to /opt, so there is no need to share `/tmp/cnao` between the provision stage and the run stage, it is done there with that copy already.

Reply:
Right, disregard
/lgtm